assets/src/blocks/amp-story-cta/index.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { G, Path, SVG } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import {
+	RichText,
+	getColorClassName,
+	getFontSizeClass,
+} from '@wordpress/editor';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+
+const schema = {
+	url: {
+		type: 'string',
+		source: 'attribute',
+		selector: 'a',
+		attribute: 'href',
+	},
+	text: {
+		type: 'string',
+		source: 'html',
+		selector: 'a',
+	},
+	backgroundColor: {
+		type: 'string',
+	},
+	textColor: {
+		type: 'string',
+	},
+	customBackgroundColor: {
+		type: 'string',
+	},
+	customTextColor: {
+		type: 'string',
+	},
+	fontSize: {
+		type: 'string',
+	},
+	customFontSize: {
+		type: 'number',
+	},
+	autoFontSize: {
+		type: 'number',
+	},
+	ampFontFamily: {
+		type: 'string',
+	},
+};
+
+export const name = 'amp/amp-story-cta';
+
+export const settings = {
+	title: __( 'Call to Action', 'amp' ),
+
+	description: __( 'Prompt visitors to take action with a button-style link.', 'amp' ),
+
+	icon: <SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><Path fill="none" d="M0 0h24v24H0V0z" /><G><Path d="M19 6H5c-1.1 0-2 .9-2 2v8c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V8c0-1.1-.9-2-2-2zm0 10H5V8h14v8z" /></G></SVG>,
```

Comment:
One idea for the CTA icon would be a “starburst”.

Reply:
Something like the 💥 emoji? you mean

Reply:
Something like this:

![image](https://user-images.githubusercontent.com/134745/55649502-fc889b00-5797-11e9-82e0-7824d089d6d0.png)

But not exactly 😄 

Actually, what you have with the rectangle does work, but what it is missing is a mouse cursor (or finger) overlaying it (e.g. 👆), to indicate that it is a button that can be clicked/touched. Since most things in AMP Stories aren't clicked, that could make its role more clear?

Reply:
That‘s what Dawid‘s suggestion was here on the PR: a rectangle with a pointer cursor.

Reply:
> <img alt="Call2Action icon" width="282" src="https://user-images.githubusercontent.com/8026823/55562124-a3ffc200-56f3-11e9-9cb8-4808acfa004a.png">

Yes! 😊 That's it.